test/Fixtures/Fixtures.yaml

```diff
@@ -0,0 +1,902 @@
+  Vendor: Wildlife Acoustics
+  StartDate: 2021-06-21T21:21:11-03:00
+  LocalStartDate: 2021-06-21T21:21:11
+  CalculatedChecksum:
+    Type:
+    Value:
+  DurationSeconds: 557056/256000
+  TotalSamples: 557056
+  Channels: 1
+  SampleRateHertz: 256000
+  BitsPerSecond: 4096000
+  BitDepth: 16
+  MemoryCard:
+    FormatType:
+    ManufacturerID:
+    OEMID:
+    ProductName:
+    ProductRevision:
+    SerialNumber:
+    ManufactureDate:
+    Speed:
+    Capacity:
+    WrCurrentVmin:
+    WrCurrentVmax:
+    WriteBlSize:
+    EraseBlSize:
+  FileLengthBytes: 1116746
+  Sensor:
+    Name: SM4BAT-FS
+    Firmware: 2.2.1
+    SerialNumber: S4U09523
+    PowerSource: 
+    Microphones:
+      - Type: U2
+        UID:
+        BuildDate:
+        Gain:
+        Channel:
+        ChannelName:
+        Sensitivity: 13.0dBFS
+    Voltage:
+    Gain:
+    Configuration:
+    BatteryLevel:
+    Temperature: 23.25C
+    LastTimeSync:
+  Location:
+    Longitude: -64.23352
+    Latitude: 45.7835
+    Altitude:
+    HorizontalAccuracy:
+    VerticalAccuracy:
+    Speed:
+    Course:
+    SampleDateTime:
+    CoordinateReferenceSystem:
+  EmbeddedChecksum:
+    Type:
+    Value:
+  EndDate:
+  BlockAlign: 2
+  Conditionals:
+    FilenameExtractor:
+      Extension: .wav
+      Stem: S4U09523_20210621_212111
+      StartDate:
+      LocalStartDate: 2021-06-21T21:21:11
+      Location:
+
+- Name: Firmware 3.3
+  FixturePath: FL_BAR_LT/3.30_NewFirmware/00008570_Dartmouth_20220331T094900-0300_Rec.flac
+  Notes:
+  Extension: .flac
+  Stem: 00008570_Dartmouth_20220331T094900-0300_Rec
+  ValidMetadata: Yes
+  MimeType: audio/flac
+  FrontierLabsLogFile: FL_BAR_LT/3.30_NewFirmware/logfile_00008570_SD1.txt
+  CanProcess:
+    - FilenameExtractor
+    - FlacHeaderExtractor
+    - FlacCommentExtractor
+    - FrontierLabsLogFileExtractor
+  Process:
+    - FilenameExtractor
+    - FlacHeaderExtractor
+    - FlacCommentExtractor
+    - FrontierLabsLogFileExtractor
+    - FLCommentAndLogExtractor
+  Vendor: Frontier Labs
+  StartDate: 2022-03-31T09:49:02-03:00
+  LocalStartDate: 2022-03-31T09:49:02
+  CalculatedChecksum:
+    Type:
+    Value:
+  DurationSeconds: 31703040/44100
+  TotalSamples: 31703040
+  Channels: 1
+  SampleRateHertz: 44100
+  BitsPerSecond: 705600
+  BitDepth: 16
+  MemoryCard:
+    FormatType: exFAT
+    ManufacturerID: 3
+    OEMID: SD
+    ProductName: SD128
+    ProductRevision: 8.5
+    SerialNumber: 1132114800
+    ManufactureDate: 2020/08
+    Speed: 50
+    Capacity: 124868608
+    WrCurrentVmin: 5
+    WrCurrentVmax: 5
+    WriteBlSize: 512
+    EraseBlSize: 65536
+  FileLengthBytes:
+  Sensor:
+    Name:
+    Firmware: 3.30
+    SerialNumber: 00008570
+    PowerSource: Ext-power
+    Microphones:
+      - Type: STD AUDIO MIC
+        UID: 009966
+        BuildDate: 2019-03-20
+        Gain: 50dB
+        Channel: 2
+        ChannelName: B
+        Sensitivity:
+    Voltage:
+    Gain:
+    Configuration:
+    BatteryLevel: 65p 3.79V
```

Comment:
Ok, already with YAML I'm seeing questionable values because it's so much easier to read in a diff.

I think we should split Battery percentage out from voltage. `BatteryLevel` should store an actual double in the range `[0.0, 1.0]` (the percentage as a unit value) and Battery voltage should be a separate field.

File this as a separate issue and address in another PR.

Reply:
There are similar issues with MemoryCard.ManufacturedDate and Microphone.Gain